src/qcasutreeitem.cpp

```diff
@@ -3,33 +3,36 @@
 QCasuTreeItem::QCasuTreeItem(QObject* parent, QString name) : QObject(parent), casuName(name), child_selected(false)
 {
     this->setData(0,Qt::DisplayRole,QStringList(casuName));
+    widget_setpoints = new QTreeWidgetItem(QStringList("Current setpoints"));
     widget_IR = new QTreeWidgetItem(QStringList("IR - Proximity"));
     widget_LED= new QTreeWidgetItem(QStringList("LED"));
     widget_temp = new QTreeWidgetItem(QStringList("Temperature"));
     widget_vibr = new QTreeWidgetItem(QStringList("Vibration"));
-    widget_setpoints = new QTreeWidgetItem(QStringList("Current setpoints"));
+
 
     //zadavanje djece IR grani:
     {
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("F"), name + ": IR - F"));
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("FL"), name + ": IR - FL"));
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("BL"), name + ": IR - BL"));
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("B"), name + ": IR - B"));
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("BR"), name + ": IR - BR"));
-        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("FR"), name + ": IR - FR"));
+        widget_IR_children.append((QTreeWidgetItem*)new QTreeBuffer(QStringList("IR - F"), name + ": IR - F"));
```

Comment:
All treeWidgetItem children have unique names because when group plotting is called, it searches correct item to plot by name (it is not selected in tree)